src/api/config/mod.rs

```diff
@@ -128,6 +128,8 @@ pub struct Config {
   pub(crate) threads: usize,
   /// Shared thread pool
   pub(crate) pool: Option<Arc<ThreadPool>>,
+  /// Number of parallel encoding slots
+  pub(crate) slots: usize,
```

Comment:
We can safely excluded this field from stable API builds.
```suggestion
  #[cfg(feature = "unstable")]
  pub(crate) slots: usize,
```